Dockerfile

```
@@ -0,0 +1,63 @@
+FROM phusion/baseimage:0.9.16
+MAINTAINER Hypothes.is Project and contributors
+
+# System dependencies
+ENV LC_ALL=C DEBIAN_FRONTEND=noninteractive
+RUN apt-add-repository ppa:brightbox/ruby-ng
+RUN apt-get -q -y update
+RUN apt-get -q -y --no-install-recommends install \
+  build-essential \
+  git \
+  libevent-dev \
+  libffi-dev \
+  libpq-dev \
+  libyaml-dev \
+  nodejs \
+  npm \
+  python-dev \
+  python-pip \
+  python-virtualenv \
+  ruby2.2 \
+  ruby2.2-dev
+
+# Provide /usr/bin/node as well as /usr/bin/nodejs
+RUN update-alternatives --install /usr/bin/node node /usr/bin/nodejs 10
+
+# Clean apt state
+RUN apt-get clean && rm -rf /var/lib/apt/lists/* /tmp/* /var/tmp/*
+
+# Base environment
+RUN pip install -qU pip virtualenv
```

I recommend not using -q (it hides errors)